doc/api/os.md

```diff
@@ -328,24 +271,13 @@ added: v0.5.0
 
 * Returns: {string}
 
-The `os.platform()` method returns a string identifying the operating system
-platform as set during compile time of Node.js.
-
-Currently possible values are:
+Returns a string identifying the operating system platform. The value is set
+at compile time. Possible values are `'aix'`, `'darwin'`, `'freebsd'`, `'linux'`, `'openbsd'`, `'sunos'`, and `'win32'`.
```

Comment:
Long line?

Reply:
I wonder why the markdown linter didn't catch that. Anyway, fixed.

Reply:
Resolved in https://github.com/nodejs/node/pull/30102/commits/a63fc230b17fde76831b9b11c1a3ae8cac9de075